lib/workload/stateless/stacks/filemanager/docs/MOVED_OBJECTS.md

```diff
@@ -0,0 +1,47 @@
+# Tracking moved objects
+
+The filemanager tracks records from S3 events, which do not describe how objects move from one location to another. Using
+S3 events alone, it's not possible to tell whether an object that has been deleted in one place and created in another is
+the same object that has been moved, or two different objects. This is because S3 only tracks `Created` or `Deleted`
+events.
+
+To track moved objects, the filemanager stores additional information in S3 tags, that gets copied when the object
```

Comment:
doc nit:

"stores additional information in S3 tags. The tag field X gets updated when the object is moved."

Reply:
and/or "see below for tag key-value details"